packages/components/phone-number/src/components/osds-phone-number/osds-phone-number.scss

```diff
@@ -1,3 +1,36 @@
 @import '~@ovhcloud/ods-common-theming/color/ods-theming-color';
 @import '~@ovhcloud/ods-common-theming/ods-theme';
 
+:host {
+  display: inline-flex;
+
+  osds-select {
```

Comment:
if we go with BEM, we should set class with correct type (element, modifier) on every element and never use generic selector.
Here your code should look like a representation of your DOM, something more like:

```
.phone-number {
  ...

  &__select {
    ...

    &__option {
      ...

      &__flag {
        ...
      }
    }
  }

  &__input {
    ...

    &--first {
      ...
    }
  }
}
```